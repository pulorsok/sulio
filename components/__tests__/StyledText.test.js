import * as React from 'react';
import renderer from 'react-test-renderer';

import { MonoText, PageTitle } from '../StyledText';

it('Page title', () => {
  const tree = renderer.create(<PageTitle>Snapshot test!</PageTitle>).toJSON();
  expect(tree).toMatchSnapshot();
})
it(`renders correctly`, () => {
  const tree = renderer.create(<MonoText>Snapshot test!</MonoText>).toJSON();
  expect(tree).toMatchSnapshot();
});
