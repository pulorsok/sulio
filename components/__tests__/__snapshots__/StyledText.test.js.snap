// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page title 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#111111",
        "fontFamily": "Arial",
        "fontSize": 48,
        "fontWeight": "bold",
        "textAlign": "left",
      },
    ]
  }
>
  Snapshot test!
</Text>
`;

exports[`renders correctly 1`] = `
<Text
  style={
    Array [
      undefined,
      Object {
        "fontFamily": "space-mono",
      },
    ]
  }
>
  Snapshot test!
</Text>
`;
